docs/src/ci.md

```diff
@@ -191,6 +243,56 @@ pipeline {
 }
 ```
 
+### Buildkite
+
+Buildkite supports [containerized builds with Docker](https://buildkite.com/docs/tutorials/docker-containerized-builds) using either their [Docker Compose plugin](https://github.com/buildkite-plugins/docker-compose-buildkite-plugin) or their [Docker plugin](https://github.com/buildkite-plugins/docker-buildkite-plugin).
```

Comment:
This is super neat—I worry about putting this in the official docs without us having tried it out yet and knowing how to triage/troubleshoot issues that arise.

Reply:
Yeah I was curious about people asking to troubleshoot these configs but looking at the `Azure` and `GitLab CI` example we add the `...` to indicate this is like a starter idea that they need to finish. Maybe there could be a warning to say this is generally the idea for how to configure each CI.

Although this is a little long and fully working example to explain a _mostly_ copy/paste example that they don't need to go all over the place to understand how to configure each, while sprinkling in some of the nuances 🧁 .
 
That said, I'm completely open to any changes y'all feel is necessary.